backend/core/scripts/minimal-listing.json

```diff
@@ -4,22 +4,20 @@
   "CSVFormattingType": "basic",
   "countyCode": "Detroit",
   "preferences": [],
-  "property": {
-    "buildingAddress": {
-      "city": "Detroit",
-      "state": "MI",
-      "street": "Main St",
-      "zipCode": "48127",
-      "latitude": "42.36",
-      "longitude": "-83.1"
-    },
-    "units": [
-      {
-        "unitType": "oneBdrm",
-        "status": "available"
-      }
-    ]
+  "buildingAddress": {
+    "city": "Detroit",
+    "state": "MI",
+    "street": "Main St",
+    "zipCode": "48127",
+    "latitude": "42.36",
+    "longitude": "-83.1"
```

Comment:
So a listing now has a buildingAddress field? And that's used to create the property?

Reply:
Strictly speaking, the `Listing` *entity* doesn't have a `buildingAddress` (or `units`) field; ie in the DB, there is no direct link from the `listings` table to either the `address` or `units` table. But Exygy has made it so that both the `ListingCreateDto` (the thing passed to the backend controller to create a listing) and the `ListingDto` (the thing returned by the backend when you `GET` one or more listings) both have `buildingAddress` and `units` populated (they're "derived" from the associated `property`). So from the perspective of the frontend, when it calls `GET /listings`, it gets back objects that do include `buildingAddress` and `units`, even though there's some magic happening in the backend to put together these fields from `Listing` and `Property`.